From f4e47a5a2c4a5da19b37468a607d2b978329d47c Mon Sep 17 00:00:00 2001
From: Ioan Sucan <isucan@gmail.com>
Date: Sat, 15 Feb 2014 20:56:12 -0800
Subject: [PATCH] fix install rules

---
 CMakeLists.txt | 12 +++++++++---
 1 file changed, 9 insertions(+), 3 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index bf3e44e..b9a0134 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -129,8 +129,14 @@ install(DIRECTORY include/ DESTINATION include
   PATTERN ".DS_Store" EXCLUDE
 )
 
-install(FILES "${pkg_conf_file}" DESTINATION lib/pkgconfig/ COMPONENT pkgconfig)
+find_package(catkin)
+catkin_package(
+  INCLUDE_DIRS include
+  LIBRARIES ${PROJECT_NAME}
+  )
 
+#install(FILES "${pkg_conf_file}" DESTINATION lib/pkgconfig/ COMPONENT pkgconfig)
 
-enable_testing()
-add_subdirectory(test)
+
+#enable_testing()
+#add_subdirectory(test)
-- 
2.7.4

